### ISSUE TITLE
SEO warning, because of <a> tag linking to an image instead of website

### ISSUE THREAD

**Issue by fatheaddrummer** [feimosi/baguetteBox.js#226]
Hi and thank you so much for the awesome plugin!

Yesterday we received a warning from SEM rush, that <a> tags that are linking to an image will be interpreted as poor architecture by search engines. We never had this issue before, but i think this could be possibly connected to latest google algorithm update?! I don't really know, but I'll post the exact warning in the end of this post. The warning has been submitted by SEM Rush. Would it be possible, to put the image source into an attribute like data-src="/path/to/src.jpg" and the href="#" ? It seems to be an issue and maybe it's worth thinking about it. This issue is totally new for me as well and I really think it has to do somthing with the google update, since I have been using your plugin for 1-2 years, but this is the first time, we receive the warning. Please let me know what you think about this. Thank you so much and have a great day!

The warning by SEM Rush: 
We detected that some links to resources are formatted with <a href> HTML element. An <a> tag with a href attribute is used to link to other webpages and must only contain a page URL. Search engines will crawl your site from page to page by following these HTML page links. When following a page link that contains a resource, for example, an image, the returned page will not contain anything except an image. This may confuse search engines and will indicate that your site has poor architecture.

**Comment by zalevsk1y** [comment]
If SEM rush confuses image links in the <a> tag, use the `rel = 'nofollow'` attribute. This will show the search engine that this link should be ignored.[GooglesBlog](https://webmasters.googleblog.com/2019/09/evolving-nofollow-new-ways-to-identify.html):

> rel="nofollow": Use this attribute for cases where you want to link to a page but don’t want to imply any type of endorsement, including passing along ranking credit to another page

**Comment by Bee406** [comment]
We ran into the same warning from SEM Rush and the nofollow attribute did not work. Would love any input as we'd really like to keep using this plugin. Thanks so much!

**Comment by whitemikenigga** [comment]
> We ran into the same warning from SEM Rush and the nofollow attribute did not work. Would love any input as we'd really like to keep using this plugin. Thanks so much!

Do u solved this issue?

**Comment by Bee406** [comment]
> > We ran into the same warning from SEM Rush and the nofollow attribute did not work. Would love any input as we'd really like to keep using this plugin. Thanks so much!
> 
> Do u solved this issue?

No, we are still having the same problem. Not a huge deal but would be interested in finding a fix.

**Comment by admench** [comment]
I am having the same issue and can confirm that adding `rel="nofollow"` does not fix the issue in semrush. Might have to move to a different lightbox library.

**Comment by LeoniePhiline** [comment]
Linking to an image is not poor architecture - it is a fallback to images without or with disabled JavaScript.
Completely best-practice across the web.

**Comment by 19890843006** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
Falai24, @sc0ttj, @zevaverbach, @DiSTaRUS, @yuqirose, @JustGreg, @ajitsinghkaler, @craiem, @onemouth, @rrkandala, @rjcoelho, @Band1to, @fabiofederici, @dagaa, @Madril, @lukehedger, @narze, @bylatt, @phntm7, @Zowie
